### DISCUSSION TITLE
Linkchecker throws Connection Error

### DISCUSSION THREAD

**Discussion by jayavenkats** [linkchecker/linkchecker#725] [Q&A]
I followed the steps to install linkchecker and run it. but i face the below error. 

linkchecker www.example.com is the command i run

**Output**
INFO linkcheck.cmdline 2023-02-15 12:42:18,145 MainThread Checking intern URLs only; use --check-extern to check extern URLs.
LinkChecker 10.2.1
Copyright (C) 2000-2016 Bastian Kleineidam, 2010-2022 LinkChecker Authors
LinkChecker comes with ABSOLUTELY NO WARRANTY!
This is free software, and you are welcome to redistribute it under
certain conditions. Look at the file `LICENSE' within this distribution.
Read the documentation at https://linkchecker.github.io/linkchecker/
Write comments and bugs to https://github.com/linkchecker/linkchecker/issues

Start checking at 2023-02-15 12:42:18+005

URL        `http://www.example.com'
Real URL   http://www.example.com
Check time 0.037 seconds
Result     Error: ConnectionError: HTTPConnectionPool(host='www.example.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000025A53914A10>: Failed to establish a new connection: [E...

Statistics:
Downloaded: 0B.
Content types: 0 image, 0 text, 0 video, 0 audio, 0 application, 0 mail and 1 other.
URL lengths: min=22, max=22, avg=22.

That's it. 1 link in 1 URL checked. 0 warnings found. 1 error found.
Stopped checking at 2023-02-15 12:42:19+005 (1 seconds)

any help would be appreciated.

**Comment by cjmayo** [comment]
The simple answer is: only check sites you are responsible for.

The web is complicated and may not want to be crawled.
